test/it/negative.spec.js

```diff
@@ -1,9 +1,16 @@
 const compiler = require("../utils/compiler");
 const listSubDirectories = require("../utils/listSubDirectories");
+
 const NEGATIVE_ROOT_PATH = "test/it/code-samples/negative";
-const negativeRoots = listSubDirectories(NEGATIVE_ROOT_PATH);
+const ignoredTests = [
+  "test/it/code-samples/negative/pages-$w-dynamic-not-on-page"
```

Comment:
Do we have a jira for fixing this test? Can you add a link pls?

Reply:
Thanks! Just add to our current sprint